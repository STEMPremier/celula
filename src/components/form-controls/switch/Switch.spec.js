/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { Switch } from '.';

describe('<Switch />', () => {
  describe('render()', () => {
    test('render the Switch component', () => {
      const wrapper = shallow(
        <Switch
          label="Switch Test Label"
          name="Switch Test Name"
          value="switch value"
        />,
      );
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
/* eslint-enable react/jsx-filename-extension */
