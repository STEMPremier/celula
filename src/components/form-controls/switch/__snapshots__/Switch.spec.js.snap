// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Switch /> render() render the Switch component 1`] = `
<div
  className="ce-switch"
>
  <input
    checked={false}
    className="ce-switch__checkbox"
    disabled={false}
    form=""
    id="Switch Test Name_switch value"
    name="Switch Test Name"
    onChange={[Function]}
    type="checkbox"
    value="switch value"
  />
  <label
    className="ce-switch__slider"
    htmlFor="Switch Test Name_switch value"
  >
    Switch Test Label
  </label>
</div>
`;
